lzero/policy/random_policy.py

```diff
@@ -50,13 +55,17 @@ def default_model(self) -> Tuple[str, List[str]]:
                 return 'EfficientZeroModel', ['lzero.model.efficientzero_model']
             elif self._cfg.type == 'muzero':
                 return 'MuZeroModel', ['lzero.model.muzero_model']
+            elif self._cfg.type == 'sampled_efficientzero':
+                return 'SampledEfficientZeroModel', ['lzero.model.sampled_efficientzero_model']
             else:
                 raise NotImplementedError("need to implement pipeline: {}".format(self._cfg.type))
         elif self._cfg.model.model_type == "mlp":
             if self._cfg.type == 'efficientzero':
                 return 'EfficientZeroModelMLP', ['lzero.model.efficientzero_model_mlp']
             elif self._cfg.type == 'muzero':
                 return 'MuZeroModelMLP', ['lzero.model.muzero_model_mlp']
+            elif self._cfg.type == 'sampled_efficientzero':
+                return 'SampledEfficientZeroModelMLP', ['lzero.model.sampled_efficientzero_modelMLP']
```

Comment:
why use the name `sampled_efficientzero_modelMLP`

Reply:
because it's in `elif self._cfg.model.model_type == "mlp":` branch.